src/test/java/org/openmrs/contrib/qaframework/page/RegistrationEditSectionPage.java

```diff
@@ -141,4 +141,9 @@ public ClinicianFacingPatientDashboardPage confirmPatient() throws InterruptedEx
 		clickOn(CONFIRM);
 		return new ClinicianFacingPatientDashboardPage(this);
 	}
+	
+	public RegistrationSummaryPage confirmPatientEdit() throws InterruptedException {
+		clickOn(CONFIRM);
```

Comment:
is CONFIRM a link or button?

Reply:
@kdaud it is a button

Reply:
how can one tell that CONFIRM is a button?